src/flag_gems/ops/copy.py

```diff
@@ -0,0 +1,9 @@
+import triton
+
+from ..utils import pointwise_dynamic
+
+
+@pointwise_dynamic(is_tensor=(True,), promotion_methods=[(0, "DEFAULT")])
+@triton.jit
+def copy_(src):
+    return src
```

Comment:
`copy_` means inplace copy in torch convention.

So I suggest wrap it in 2 functions.

`def copy(src: Tensor) -> Tensor`
`def copy_(out: Tensor(a), src: Tensor) -> Tensor(a)`

Otherwise, we may not know it can take output parameter. (This is a common issue with PointwiseDynamicFunction.)

Reply:
I used it as an inplace copy but the function actually can do both inplace and out of place.. let's rename it as copy.